StandaloneMmPkg/Drivers/StandaloneMmIplPei/StandaloneMmIplPei.inf

```diff
@@ -30,15 +30,24 @@
   MdePkg/MdePkg.dec
   MdeModulePkg/MdeModulePkg.dec
   StandaloneMmPkg/StandaloneMmPkg.dec
+  UefiCpuPkg/UefiCpuPkg.dec
 
 [LibraryClasses]
   PeimEntryPoint
+  PeiServicesTablePointerLib
+  PeiServicesLib

```

Comment:
It seems like PeiServicesTablePointerLib and PeiServicesLib are not needed in this commit

Reply:
Removed it in new commit